cmd/soroban-rpc/main.go

```diff
@@ -62,6 +74,37 @@ func main() {
 			},
 			Usage: "minimum log severity (debug, info, warn, error) to log",
 		},
+		&config.ConfigOption{
```

Comment:
I created separate ticket on quickstart to update it's config on soroban runtime, which will now need to include some of these new settings:
https://github.com/stellar/quickstart/issues/406

quickstart won't be able to run this version of rpc until then.

Reply:
👍 thanks!